app/src/main/java/org/mifos/mobile/ui/fragments/AccountFragment.kt

```diff
@@ -0,0 +1,19 @@
+package org.mifos.mobile.ui.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+
+class AccountFragment : BaseFragment() {
```

Comment:
Are you planning to implement these Fragments, as a part of other PR? If yes, add proper javadoc at the top of the class with related info. (applies for other places too)

Reply:
Done.